README.md

```diff
@@ -79,6 +72,11 @@ Pylance provides users with the ability to customize their Python language suppo
     -   Used to specify extra search paths for import resolution. This replaces the old `python.autoComplete.extraPaths` setting.
     -   Default value: empty array
 
+-   `python.analysis.indexing`
+
+    - Used to specify whether Pylance should index installed third party libraries and user files to provide features such as auto-import, add import, workspace symbols, etc.
+    - Default value: null
```

Comment:
should we list all possible value and show which one is default? like `python.analysis.useLibraryCodeForTypes` and others?

Reply:
Great suggestion, done!